transpire/cli/image.py

```diff
@@ -10,7 +12,45 @@ def commands(**kwargs):
 @commands.command()
 def build(**kwargs):
     """build defined images"""
-    raise NotImplementedError("Not yet implemented!")
+
+    # ensure we have buildkit installed
+    if shutil_which("buildctl") is None:
+        raise RuntimeError("You must install Buildkit to use this script.")
+
+    # get remote and additional tags (like one by commit)
+    remote: str  = kwargs.get('registry', None)
+    tags: list = kwargs.get('additional_tags', [])
+
+    # read the dockerfile path, app name, tags, context
+    with open(f".transpire.toml", 'r') as tomlFile:
+        config = parse_toml(tomlFile.read())
+        
+        build_data = config['build'][0]
```

Comment:
We want to loop over all the items in the `build` list and build all of them, not just the first one!

Reply:
I've not worked too much with toml so I don't know what multiple build targets in a single toml file look like. I went off of the list the parser gives and integrate over `config['build']`.